go.mod

```diff
@@ -1,19 +1,21 @@
 module github.com/hashicorp/packer-plugin-scaffolding
 
-go 1.20
+go 1.21.0
+
+toolchain go1.22.6
```

Comment:
I would suggest we leave `toolchain` out here, as otherwise anyone wanting to build the project will need at least this exact version of the go toolchain, which kind of contradicts what `go` specifies.

Reply:
Addressed in https://github.com/hashicorp/packer-plugin-scaffolding/commit/1eaaccbcc5e2ebe4908a7326e1e6162297a29441.